js/freewheel.js

```diff
@@ -429,10 +429,10 @@ OO.Ads.manager(function(_, $) {
           } else {
             fwContext.setParameter("renderer.video.clickDetection", true, tv.freewheel.SDK.PARAMETER_LEVEL_GLOBAL);
             slotStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId) {
-                amc.notifyNonlinearAdStarted(adId);
+                amc.sendURLToLoadAndPlayNonLinearAd(ad, ad.id, null);
               }, this, ad.id);
             slotEndedCallbacks[ad.ad.getCustomId()] = _.bind(function(adId) {
-                amc.notifyNonlinearAdEnded(adId);
+                amc.sendURLToLoadAndPlayNonLinearAd(ad, ad.id, null);
```

Comment:
You don't want to change that here :), thisis that the linear ad ended, diferent callback, please revert this line